src/Microsoft.CodeQuality.Analyzers/CSharp/Performance/CSharpDoNotUseCountWhenAnyCanBeUsed.Fixer.cs

```diff
@@ -0,0 +1,175 @@
+    /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public sealed class CSharpDoNotUseCountWhenAnyCanBeUsedFixer : DoNotUseCountWhenAnyCanBeUsedFixer
+    {
+        /// <summary>
+        /// Tries the get a fixer the specified <paramref name="node" />.
+        /// </summary>
+        /// <param name="node">The node to get a fixer for.</param>
+        /// <param name="expression">If this method returns <see langword="true" />, contains the expression to be used to invoke <c>Any</c>.</param>
+        /// <param name="arguments">If this method returns <see langword="true" />, contains the arguments from <c>Any</c> to be used on <c>Count</c>.</param>
+        /// <param name="negate">If this method returns <see langword="true" />, indicates whether to negate the expression.</param>
+        /// <returns><see langword="true" /> if a fixer was found., <see langword="false" /> otherwise.</returns>
+        protected override bool TryGetFixer(SyntaxNode node, out SyntaxNode expression, out IEnumerable<SyntaxNode> arguments, out bool negate)
+        {
+            if (node.IsKind(SyntaxKind.InvocationExpression))
+            {
+                GetFixerForEqualsMethod((InvocationExpressionSyntax)node, out expression, out arguments);
+                negate = true;
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.EqualsExpression))
+            {
+                GetFixerForEqualityExpression((BinaryExpressionSyntax)node, out expression, out arguments);
+                negate = true;
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.NotEqualsExpression))
+            {
+                GetFixerForEqualityExpression((BinaryExpressionSyntax)node, out expression, out arguments);
+                negate = false;
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.LessThanExpression))
+            {
+                GetFixerForLessThanExpression((BinaryExpressionSyntax)node, out expression, out arguments, out negate);
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.LessThanOrEqualExpression))
+            {
+                GetFixerForLessThanOrEqualExpression((BinaryExpressionSyntax)node, out expression, out arguments, out negate);
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.GreaterThanExpression))
+            {
+                GetFixerForGreaterThanExpression((BinaryExpressionSyntax)node, out expression, out arguments, out negate);
+                return true;
+            }
+            else if (node.IsKind(SyntaxKind.GreaterThanOrEqualExpression))
+            {
+                GetFixerForGreaterThanOrEqualExpression((BinaryExpressionSyntax)node, out expression, out arguments, out negate);
+                return true;
+            }
+
+            expression = default;
```

Comment:
Is this code path possible? I understand if this is just a defensive approach...

Reply:
It's defensive. Just in case. 😄